frontend/src/pages/projects/screens/spawner/storage/ClusterStorageTable.tsx

```diff
@@ -97,7 +98,13 @@ export const ClusterStorageTable: React.FC<ClusterStorageTableProps> = ({
                 spaceItems={{ default: 'spaceItemsSm' }}
               >
                 <FlexItem>
-                  <Truncate content={row.name} />
+                  <Truncate
+                    content={
+                      row.existingPvc?.metadata.name === row.name
```

Comment:
what is this check for? couldnt you just do 

```tsx
row.existingPvc ? getDisplayNameFromK8sResource(row.existingPvc) : row.name

Reply:
This condition is no longer needed I have reverted back to original one
(based on current implementation